### PR TITLE
docs(35/docker) fix network instructions

### PR DESCRIPTION
**Pull request by Tieske**
This is an addition to #1392

### CODE REVIEW

File: app/enterprise/0.35-x/deployment/installation/docker.md [hunk 1]
```diff
@@ -45,13 +45,24 @@ with the URL you received in your welcome email:
 
     (Replace "IMAGE ID" with the one matching your repository, seen in step 4)
 
-6. Start your database
+6. Create a Docker network
+
+    You will need to create a custom network to allow the containers to discover
+    and communicate with each other. In this example `kong-ee-net` is the network name,
```
Comment (gszr) [inline app/enterprise/0.35-x/deployment/installation/docker.md]: `In this example, ...`

File: app/enterprise/0.35-x/deployment/installation/docker.md [hunk 2]
```diff
@@ -45,13 +45,24 @@ with the URL you received in your welcome email:
...
+    you can use any name.
```
Comment (gszr) [inline app/enterprise/0.35-x/deployment/installation/docker.md]: `but you can use any name`

Comment (h3xar0n) [reply app/enterprise/0.35-x/deployment/installation/docker.md]:
```suggestion
    but you can use any name.
```

File: app/enterprise/0.35-x/deployment/installation/docker.md (hunk 1 above)
Comment (h3xar0n) [reply app/enterprise/0.35-x/deployment/installation/docker.md]:
```suggestion
    and communicate with each other. In this example, `kong-ee-net` is the network name,
```

**Comment by Tieske** [comment]
lgtm!

**Comment by Tieske** [comment]
@h3xar0n what is this PR waiting for? can we merge it?

**Pull request merged**